src/app/home/home.component.ts

```diff
@@ -10,32 +12,38 @@ import {PageMetaService} from '../page-meta.service';
 export class HomeComponent implements OnInit {
   mainTitle = 'Matching Donations.\nMultiplying Impact.';
 
-  showMNFNowOpen: boolean;
-  showGreenMatchFund: boolean;
-  showChampionsForChildren: boolean;
-  showExplore: boolean;
+  highlightCards: readonly highlightCard[];
 
   public constructor(private pageMeta: PageMetaService) {}
 
   ngOnInit() {
+    this.highlightCards = [
+      {
+        headerText: 'Turkey-Syria Earthquake Appeal',
+        bodyText: "Double the impact of your donation",
+        iconColor: "brand-4",
+        backgroundImageUrl: new URL('/assets/images/emergency-card.png', environment.donateUriPrefix),
+        button: {
+          text: "Donate now",
+          href: new URL('/turkey-syria-earthquake-appeal', environment.donateUriPrefix),
+        }
+      },
+      {
+        headerText: 'One donation. Twice the impact.',
+        bodyText: "You donate.\nWe double it.",
+        iconColor: "primary",
+        backgroundImageUrl: new URL('/assets/images/blue-texture.jpg', environment.donateUriPrefix),
+        button: {
+          text: "Explore now",
+          href: new URL('/explore', environment.donateUriPrefix),
+        }
+      },
+    ];
+
```

Comment:
I suppose while this is hard-coded it could be in constructor instead of ngOnInit. But I don't think it matters much, and it would just have to move back to ngOnInit when it starts using a service call to get the data.